app/src/androidTest/java/br/com/simplepass/loadingbutton/ViewVisibilityIdlingResource.kt

```diff
@@ -0,0 +1,40 @@
+package br.com.simplepass.loadingbutton
+
+import android.view.View
+import androidx.test.espresso.IdlingResource
+import androidx.test.espresso.IdlingResource.ResourceCallback
+
+/*
+    IdlingResource that waits until the specified view matches the expected visibility criteria
+    Converted to Kotlin from original implementation found here: https://stackoverflow.com/a/35080198
+ */
+class ViewVisibilityIdlingResource(private val view: View, private val expectedVisibility: Int) : IdlingResource {
+
+    private var mIdle: Boolean = false
```

Comment:
This library is not using Hungarian notation anymore. Just `idle` would be better =]

Reply:
Copy/paste :) I'll update.